package.json

```diff
@@ -19,6 +18,7 @@
     "@reach/dialog": "^0.10.3",
     "@reach/portal": "^0.10.3",
     "@react-hook/window-scroll": "^1.3.0",
+    "@reduxjs/toolkit": "^1.5.0",
```

Comment:
why did we need this?

Reply:
this was automatic from installation, the change is just a line positioning, it didn't update anything